ledger/ledger-utxo/src/main/java/net/corda/v5/ledger/utxo/UtxoLedgerService.java

```diff
@@ -185,4 +186,53 @@ void sendUpdatedTransactionBuilder(
             @NotNull UtxoTransactionBuilder transactionBuilder,
             @NotNull FlowSession session
     );
+
+
+    /**
+     * Creates a query object for a named ledger query with the given name - if it exists. This query can be executed later.
```

Comment:
Remove the `if it exists` because it implies that it does nothing if it doesn't exist, instead it will error later on and throw in the user code.